packages/webapp/pages/onboarding.tsx

```diff
@@ -156,14 +157,21 @@ export function OnboardPage(): ReactElement {
       });
     }
 
+    const { id, defaultValue } = feature.userAcquisition;
+    const userAcquisitionVersion =
+      growthbook?.getFeatureValue(id, defaultValue) ?? UserAcquisition.Control;
```

Comment:
This could just be `useFeature(` no? 
It handles all this magic internally.

Reply:
That seems like a engineering decision.
But yes in that case rather have a hook, or adjust the hook or something.

Basically the whole idea is that it handles the fallback itself, so actually not sure if it won't do the same already?

This is also conditional of the gb value, thus enrolls?

Reply:
Or do we automatically enroll the user when GrowthBookContext is involved already? Sorry, I don't really understand how all of this works :)

Reply:
AFAIK when you want to retrieve a feature value for someone.

This is the function we have:
```js
export function useFeatureValue<T extends JSONValue = any>(
  id: string,
  fallback: T
): WidenPrimitives<T> {
  const growthbook = useGrowthBook();
  return growthbook
    ? growthbook.getFeatureValue(id, fallback)
    : (fallback as WidenPrimitives<T>);
}
```

Thus would pretty much do the same no?